Week-3/assignment3_3.js

```diff
@@ -0,0 +1,24 @@
+function createIncrement() {
+    let count=0;
+    function increment() {
+        count++;
+    }
+    let message=`Count is${count}`;
+    
+    function log() {
+        console.log(message);
+    }
+    return[increment,log];
+}
+const [increment,log] = createIncrement();
```

Comment:
How this line number 13 is working please explain ?

Reply:
We are destructuring the array here